website/src/pages/community/promotion.js

```diff
@@ -0,0 +1,211 @@
+import React from 'react'
+import Layout from '@theme/Layout'
+import styles from './promotion.module.css'
+
+const promotionRules = [
+  {
+    role: 'Reviewer',
+    icon: '👀',
+    level: 1,
+    requirements: 'Active contributions within one release cycle',
+    details: [
+      'Commits and reviews in workgroups',
+      'Active participation in project activities',
+      'Help maintain code quality',
+      'Engage in community discussions',
+    ],
+    permissions: 'Triage Permission',
+    timeline: 'After each release (2-3 month intervals)',
+    application: 'Must be nominated by a maintainer',
+    color: '#4CAF50',
+  },
+  {
+    role: 'Committer',
+    icon: '💻',
+    level: 2,
+    requirements: 'Sustained contributions across two consecutive releases',
+    details: [
+      'Major feature development in workgroups',
+      'Significant refactoring contributions',
+      'Performance optimization improvements',
+      'Demonstrate technical leadership',
+      'Mentor new contributors',
```

Comment:
`Review open PRs`
`Help open GitHub Issues`
`Engage in community meetings and slack channel discussions`